### PR TITLE
WRN-12124: Remove deprecated babel-eslint module

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
`babel-eslint` has been deprecated

```
`babel-eslint` is now `@babel/eslint-parser`. This package will no longer receive updates.
```

### Resolution
Remove deprecated `babel-eslint` 

### Links
WRN-12124

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Would you please take a look at the travis errors?

**Comment by hong6316** [comment]
> Would you please take a look at the travis errors?

@MikyungKim 
for fixing travis errors, we need to release `eslint-plugin-enact` & `eslint-config-enact`.
please check this order: http://clm.lge.com/issue/browse/WRN-12124?focusedCommentId=3882191&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3882191

> 0:0  error  Parsing error: Cannot find module 'babel-preset-react-app/prod'

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "webdriverio": "^7.9.0"
   },
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
+    "@babel/eslint-parser": "^7.16.3",
```
Comment (MikyungKim) [inline package.json:42]:
```suggestion
    "@babel/eslint-parser": "^7.17.0",
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
For travis fail, it should work when `eslint-config-enact` package is updated.

**Pull request merged**